packages/extension-onboarding/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@snickerdoodlelabs/extension-onboarding",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
This version isn't used for anything, FYI

Reply:
Good to know